### PR TITLE
add translate function for folder_contents page [terapyon]

### PR DESCRIPTION
**Pull request by terapyon**

### CODE REVIEW

File: CHANGES.rst [hunk 1]
```diff
@@ -6,7 +6,7 @@ Changelog
 
 Breaking changes:
 
-- *add item here*
+- add translate function for folder_contents page [terapyon]
```
Comment (vincentfretin) [inline CHANGES.rst]: You should put the entry in "Bug fixes", not "Breaking changes"

**Comment by terapyon** [comment]
@vincentfretin  Thank you for reviewing.

**Comment by jensens** [comment]
Hence there was no activity for a long time, I propose to close this PR within next two weeks. If you do not feel ok with this, please speak up and provide us a rough schedule.

**Comment by jensens** [comment]
There was no activity on this for a while. I close this request for now. If you plan to work further on this please reopen and go on.

**Pull request closed without merging**